importer/SeShipSv.py

```diff
@@ -118,6 +119,13 @@ def set_monuments_all_id(self):
         """
         self.monuments_all_id = self.signal
 
+    def set_descriptions(self):
+        """Set default descriptions."""
+        DESC_BASES = {"sv": "K-märkt fartyg",
```

Comment:
I believe we can clarify this further by saying "Kulturmärkt bruksfartyg" [1] to distinguish it even more from "Kulturmärkta fritidsbåtar" [2]. (Both of which are currently mentioned on [3] which is the wp page for the protection status)

[1] https://sv.wikipedia.org/wiki/Lista_%C3%B6ver_kulturm%C3%A4rkta_bruksfartyg_i_Sverige
[2] https://sv.wikipedia.org/wiki/Lista_%C3%B6ver_k-m%C3%A4rkta_fritidsb%C3%A5tar
[3] https://sv.wikipedia.org/wiki/Kulturm%C3%A4rkning_av_fartyg_i_Sverige